tasks/InsightAppSec/task.json

```diff
@@ -102,7 +102,11 @@
             "label": "Vulnerability Query",
             "required": false,
             "helpMarkDown": "If the scan results contain vulnerabilities that match this query, then the task will fail.",
-            "visibleRule": "hasScanGating = true"
+            "visibleRule": "hasScanGating = true",
+            "validation": {
+                "expression": "isMatch(value, '^(?!.*\").*$', 'IgnoreCase')",
+                "message": "Double quote (\") is an invalid character, use single quote (')."
+            }
```

Comment:
This regex will disallow the double quote char ". Ideally we would have a regex that guarantees the correctness of the Vulnerability Query, however I don't know of the full list of valid characters/formats and im reluctant to add strict validation where there was none before, possibly breaking customers who are already using this integration.

Reply:
I like this 👍🏻

Reply:
Agreed, I think this is likely one of the more common mistakes people make when entering a query. Probably not worth strictly validating now since I'm pretty sure they've expanded the query options and introduced more complexity